polaris/tasks/ocean/merry_go_round/forward.yaml

```diff
@@ -0,0 +1,78 @@
+ocean:
+  time_management:
+    config_stop_time: none
+    config_run_duration: {{ run_duration }}
+  time_integration:
+    config_dt: {{ dt }}
+    config_time_integrator: {{ time_integrator }}
+
+mpas-ocean:
+  run_modes:
+    config_ocean_run_mode: forward
+  decomposition:
+    config_block_decomp_file_prefix: graph.info.part.
+  cvmix:
+    config_use_cvmix: false
+  ALE_vertical_grid:
+    config_vert_coord_movement : fixed
+  debug:
+    config_disable_thick_all_tend: true
+    config_disable_thick_hadv: true
+    config_disable_thick_vadv: true
+    config_disable_thick_sflux: true
```

Comment:
```suggestion
```
If you have `config_disable_thick_all_tend`, you don't need the other `config_disable_thick_*` options because they are all effectively true via the first one.